tests/tape/interfaces/test_qnode_torch.py

```diff
@@ -527,6 +528,149 @@ def circuit():
         assert isinstance(res[0], torch.Tensor)
         assert isinstance(res[1], torch.Tensor)
 
+    def test_hessian(self, dev_name, diff_method, mocker, tol):
+        """Test hessian calculation of a scalar valued QNode"""
+        if diff_method not in {"parameter-shift", "backprop"}:

```

Comment:
The torch interface doesn't yet support backprop, but this should be possible soon (since Torch1.8 supports differentiating complex matrix multiplication, see #929).

Note that this class is not parametrized over `diff_method="backprop"`, so that case never occurs here. I left in the `if` statement just as a reminder that `backprop` should also be tested once #929 is merged.

Do you think we should delete this if statement for now?

Reply:
Ok, so in this test we'll currently not see `diff_method="backprop"`, but hopefully will soon? Kind of feels like the case where we should set up a separate test fail using `"backprop"` and then mark that as a strict xfail.

Reply:
Can we set the xfail flag _inside_ an if statement?